### PR TITLE
Feature/recipe story fixes

### PR DESCRIPTION
**Pull request by sergeyvlakh**
._.

### CODE REVIEW

File: src/atoms/FrontHeading.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const FheadStyle = props => css`
     white-space: nowrap;
 	overflow: hidden;
 	text-transform: uppercase;
+	letter-spacing: 0.5px;
```
Comment (evoX0) [inline src/atoms/FrontHeading.js]: change px to rem

File: src/atoms/MainRecipe/IngredientsSection.js [hunk 2]
```diff
@@ -32,11 +32,11 @@ const Counter = Styled.span`
 `;
 
 const Pers = Styled.span`
-	font-family: Roboto;
 	font-size: 1.8rem;
 	color: #333333;
 	font-weight: normal;
 	margin-left: 1.4rem;
+	letter-spacing: 1px;
```
Comment (evoX0) [inline src/atoms/MainRecipe/IngredientsSection.js]: px -> rem

File: src/atoms/MainRecipe/IngredientsSection.js [hunk 3]
```diff
@@ -46,6 +46,10 @@ const AddCart = Styled.div`
 		margin-right: 1.0rem;
 		cursor: pointer;
 	}
+	> span {
+		letter-spacing: 1px;
```
Comment (evoX0) [inline src/atoms/MainRecipe/IngredientsSection.js]: px -> rem

File: src/atoms/BylineWithTwoLines.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ const BylineImage = Styled.img.attrs({
 const TextBlock = Styled.div`
     float: left;
     color: ${props => props.theme.colors.type};
-    font-family: 'Roboto';
+    font-family: ${props => props.theme.mainFont};
```
Comment (misund) [inline src/atoms/BylineWithTwoLines.js]: I guess you are aiming for props.theme.variables.mainFont?

File: src/atoms/MainRecipe/IngredientsSection.js (hunk 3 above)
Comment (misund) [inline src/atoms/MainRecipe/IngredientsSection.js]: Px to rem

File: src/themes/mat/variables.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 export default {
-	mainFont: '"Roboto","Helvetica",Helvetica,Arial,sans-serif',
+	mainFont: '"Cabin", "Roboto","Helvetica",Helvetica,Arial,sans-serif',
```
Comment (misund) [inline src/themes/mat/variables.js]: Please confirm that what you want to do is to change the body text away from Roboto

File: src/atoms/BylineWithTwoLines.js (hunk 4 above)
Comment (sergeyvlakh) [reply src/atoms/BylineWithTwoLines.js]: Jeez, right!

File: src/themes/mat/variables.js (hunk 5 above)
Comment (sergeyvlakh) [reply src/themes/mat/variables.js]: Yuuuuup!

**Pull request merged**